triage/triaging.rst

```diff
@@ -28,6 +28,22 @@ Checklist for Triaging
 
 .. note:: Some of these fields can only be set/edited by core developers.
```

Comment:
NB, this note and the comment above related to the below are now obsolete and can be removed now that python/core-workflow#460 is implemented—but maybe scope creep on this PR.